Tests/Units/Twig/Extension/FileManagementExtension.php

```diff
@@ -23,9 +23,7 @@ public function testGetFilters()
             ->if($testedClass = $this->createTestedClassInstance())
             ->then
                 ->array($testedClass->getFilters())
-                    ->hasKey('filepath')
-                    ->hasKey('imagepath')
-                    ->hasKey('fileUrl')
+                    ->hasSize(4)
```

Comment:
If you're not test the array keys, how can you know that the returned array is what you want ?